src/components/ToolMenu/LayerTree/LoadingIndicator/index.less

```diff
@@ -0,0 +1,74 @@
+.loading-mask {
```

Comment:
Not entirely sure, but this class name _might_ conflict with others, e.g. from antd (and it's not a mask in particular).

Reply:
i have changed it to "loading-animation"

Reply:
That's still quite generic, probably something like `layer-node-title-loading-animation`?